src/rhsmlib/facts/hwprobe.py

```diff
@@ -157,7 +159,8 @@ def get_distribution(self):
             data = {
                 'PRETTY_NAME': 'Unknown',
                 'NAME': distname,
-                'ID': 'Unknown',
+                'ID': id,
+                'ID_LIKE': id_like,
```

Comment:
this must be an empty string though, as it will be replaced by the string value of the `ID_LIKE=` key